pkg/operator/operator.go

```diff
@@ -256,14 +259,16 @@ func (c *vSphereProblemDetectorController) checkForDeprecation(clusterInfo *util
 		}
 	}
 
-	_, vcenterAPIVersion := clusterInfo.GetVCenterVersion()
-	hasMinimum, err := isMinimumVersion(minVCenterVersion, vcenterAPIVersion)
-	if err != nil {
-		klog.Errorf("error parsing vcenter version: %v", err)
-	}
+	for _, vCenter := range clusterInfo.GetVCenterHostnames() {
```

Comment:
This looks as a regression: before this PR we always checked `vcenterAPIVersion`, and now we don't (outside unit tests). We don't because `clusterInfo.GetVCenterHostnames()` is always empty list, the PR populates it only for tests (see who calls `MakeClusterInfo()`).

Reply:
Good catch.  Not sure what happened to my setting loop.  let me check my history and fix this.